pkg/skaffold/plugin/shared/server.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package shared
+
+import (
+	"context"
+	"io"
+	"net/rpc"
+	"os"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/tag"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	plugin "github.com/hashicorp/go-plugin"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	yaml "gopkg.in/yaml.v2"
+)
+
+// Here is an implementation that talks over RPC
+type BuilderRPC struct {
+	client *rpc.Client
+}
+
+func (b *BuilderRPC) Init(opts *config.SkaffoldOptions, env *latest.ExecutionEnvironment) {
+	// We don't expect a response, so we can just use interface{}
+	var resp interface{}
+	args := InitArgs{
+		Opts: opts,
+		Env:  env,
+	}
+	b.client.Call("Plugin.Init", args, &resp)
+}
+
+func (b *BuilderRPC) Labels() map[string]string {
+	var resp map[string]string
+	err := b.client.Call("Plugin.Labels", new(interface{}), &resp)
+	if err != nil {
+		// You usually want your interfaces to return errors. If they don't,
```

Comment:
nit: comment is a little strange :)

Reply:
lol oops, copied & pasted from the plugin sample :sweat_smile: